datahub-web-react/src/app/permissions/policy/ManagePolicies.tsx

```diff
@@ -170,7 +170,7 @@ export const ManagePolicies = () => {
         variables: {
             input: {
                 start,
-                count: pageSize,
+                count: pageSize + 10,
```

Comment:
Not sure about this @chriscollins3456 - do you have any recollection?

Reply:
oh goodness no lol but I think you committed it in this commit? https://github.com/chriscollins3456/datahub/commit/d98cf774c019cac4115a9fd5fd4c57818794091e#diff-233e1f29c0ac1da442f3d66913b74671aa3416041e03286950edfcc82ff582b2

maybe just for testing purposes?

Reply:
Must have been but for the life of me I cannot remember why